interface/patient_file/summary/add_edit_issue.php

```diff
@@ -806,8 +808,18 @@ function divclick(cb, divid) {
                     </div>
                 </div>
                 <!-- End of reaction -->
+                <!-- Verification Status for Medication Allergy -->
+                <div class="form-group" id='row_verification'>
+                    <label class="col-form-label col-4" for="form_verification"><?php echo xlt('Verification Status'); ?>:</label>
+                    <div class="col-10">
+                        <?php
+                            echo generate_select_list('form_verification', 'allergyintolerance-verification', $irow['verification'], '', '', '', '');
+                        ?>
+                    </div>
+                </div>
+                <!-- End of Verification Statu -->
                 <div class="form-group"id='row_referredby'>
-                    <label class="col-form-label col-2" for="form_referredby"><?php echo xlt('Referred by'); ?>:</label>
+                    <label class="col-form-label col-4" for="form_referredby"><?php echo xlt('Referred by'); ?>:</label>
```

Comment:
This is actually incorrect. The Bootstrap 4 grid system goes all the way up to col-12, not col-14.